src/utils/body.ts

```diff
@@ -64,7 +64,16 @@ export async function readBody<T=any> (event: H3Event): Promise<T> {
   const body = await readRawBody(event) as string;
 
   if (event.node.req.headers["content-type"] === "application/x-www-form-urlencoded") {
-    const parsedForm = Object.fromEntries(new URLSearchParams(body));
+    const form = new URLSearchParams(body);
+    const parsedForm: Record<string, any> = {};
+    for (const [key, value] of form.entries()) {
+      if (key in parsedForm && !Array.isArray(parsedForm[key])) {
+        parsedForm[key] = [parsedForm[key]];
+        parsedForm[key].push(value);
```

Comment:
I think we need to handle when key is there and value is already an array (to test, adding another prop with same name)

Reply:
I don't think we support arrays at all, at the moment. So I think this is safe.

Reply:
```js
[...new URLSearchParams('a[]=j').entries()]
// [ "a[]", "j" ]
[...new URLSearchParams('a=[j]').entries()] 
// // [ "a", "[j]" ]
```